CONTRIBUTING.md

```diff
@@ -108,7 +108,7 @@ executable will be in your current directory and can be run as shown in the
 
 + Run the test suite locally:
 
-  `go generate ./pkg/... && make check test testrace`
+  `make generate && make check test testrace`
```

Comment:
Also not sure this needs to discuss the UI tests - that probably belongs in that package's README.